src/app/icd/server/ICDManager.h

```diff
@@ -162,7 +162,7 @@ class ICDManager : public ICDListener
     bool CheckInMessagesWouldBeSent();
 #endif // CHIP_CONFIG_ENABLE_ICD_CIP
 
-    KeepActiveFlags mKeepActiveFlags{ 0 };
+    KeepActiveFlags mKeepActiveFlags;
```

Comment:
```suggestion
    KeepActiveFlags mKeepActiveFlags{};
```

Reply:
BitFlags has a default constructor that sets it to 0, this is why I removed the constructor. I can keep it as is if you think this is important, however I believe this is not needed.